qcschema/dev/wavefunction/result_wavefunction.py

```diff
@@ -0,0 +1,78 @@
+"""
+The primary specified return  wavefunction quantities.
+"""
+
+result_wavefunction = {}
+
+# Orbitals
+result_wavefunction["return_result_orbitals_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_orbitals_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+# Density
+result_wavefunction["return_result_density_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin density in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_density_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin density in the AO basis",
```

Comment:
`return_result` here could be rough if we think about what if an energy/gradient for MP2 is specified. An energy might return SCF densities while MP2 might return relaxed or unrelaxed densities.

Reply:
I think that return_result will only be populated if the calculation explicitly asks for wavefunction results to be calculated and returned, in which case it will be unambiguous as to which is the return_result. How the wavefunction calculation is requested is [beyond the scope of this PR](https://github.com/MolSSI/QCSchema/issues/63#issuecomment-507381679).

Reply:
TODO: remove return_result prefix on these names